plugins/fresco-pd/fu-fresco-pd-common.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2020 Fresco Logic
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-fresco-pd-common.h"
+
+gchar *
+fu_fresco_pd_version_from_buf (const guint8 ver[4])
+{
+	if (ver[3] == 1 || ver[3] == 2)
```

Comment:
so the last byte is indicating order?  It seems like this version buffer is really an information blob that contains customer ID; endian order and a firmware version.  Perhaps this whole string shouldn't be exported to the `FuDevice`.  What normally shows up in release notes?  Just the 2 bytes for major and minor version?

Reply:
Nah, they use it as a complete version -- e.g. `Panther_FL7102_TypeC2HDMI_DELL_V7.122.4.0_2019_1112_1652.bin` -- my guess would be it's a typo in the silicon VHDL somewhere.

Reply:
>  Do you know what device

I have literally no idea, sorry.

Reply:
OK if you hear something from them about it let me know.  I'm not aware any PD controller that Dell uses from this company; especially a PD controller with a USB interface.